### ISSUE TITLE
User avatar image cannot be found in header

### ISSUE THREAD

**Issue by dphogit** [dphogit/my-spotify#8]
**Describe the bug**
The user's avatar image does not display (cannot be found) in the app header. It displays the alt text instead.

**To Reproduce**
Login to app and see the bug on RHS of app header (top right of screen)

**Expected behavior**
Show the user's spotify profile picture.

**Screenshots**
![image](https://user-images.githubusercontent.com/76410203/221399510-e8e37170-7c18-4ed6-ae91-5dbc766a5cdb.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Edge
 - Version: 110.0.1587.50

**Additional context**
Most likely how `next.config.js` is setup regarding image remote patterns. At the moment the hostname is set to `scontent.fmaa10-1.fna.fbcdn.net` which is a hard coded value. This worked in local environment but in prod environment the url has changed to `scontent-mia3-1.xx.fbcdn.net`. 

This needs to be generalized to handle more than just accounts that spotify created via facebook because it seems to be from a facebook CDN as well.
